### ISSUE TITLE
9.2.0 Unable to use wildcard for gcc

### ISSUE THREAD

**Issue by ssoher** [jmeubank/tdm-gcc#12]
`gcc *.c` produces the below errors, even though there are c files in the current directory

```
gcc: error: *.c: Invalid argument
gcc: fatal error: no input files
compilation terminated.
```

**Comment by jmeubank** [comment]
Hi @ssoher ,

Could you clarify for me real quick what the expected output of the command is? Just need to make sure that any fix I work on will meet your needs; this is not a typical use case.

**Comment by ssoher** [comment]
`*` is the wildcard, `gcc *.c` should compile all the c files in the current directory (or the path given as in X:\path\to\project\src\*.c). Let's say I have 3 files `main.c` `mesh.c` and `material.c`. To compile my source files, instead of calling `gcc main.c mesh.c material.c`, `gcc *.c` should give me the same result.

it seems that it's a shortcoming of the windows command prompt where it doesn't process the wildcard and defers it to the application to deal with. before tdm I was using mingw64 and the gcc in it worked fine with the wildcard. please see https://stackoverflow.com/a/45661264

my use-case is, I build my projects with a `build.bat` batch file and I don't want to explicitly write each and every C file I want to compile. I also don't want to use any build systems including make. so it's very convenient for me to pass the asterisk there.

**Comment by jmeubank** [comment]
This is something that used to work, but isn't well supported (most multi-file compilations use build systems). It should be able to be fixed in a new TDM-GCC release.

**Comment by jmeubank** [comment]
This will be fixed in the upcoming TDM-GCC 10.3.0 release.
